front/src/applications/stdcm/types.ts

```diff
@@ -22,16 +22,30 @@ export type StdcmV2SuccessResponse = Omit<
 > & {
   simulation: Extract<SimulationResponse, { status: 'success' }>;
   rollingStock: LightRollingStock;
+  creationDate: Date;
+  speedLimitByTag: string | undefined;
```

Comment:
(I prefer like this if it's ok for you :) )
```suggestion
  speedLimitByTag?: string;
```

Reply:
fix